hicommonwealth/commonwealth
# 6816 pasting content bug

## Link to Issue
Closes: #6816 

## Description of Changes
- Changed the state update to include any current items in the editor

## "How We Fixed It"

- Added a spread operator to include the old `contentDelta.ops` and add to rather than replace

## Test Plan
- Go to any community
- Make a thread or comment
- Type in some content
- Paste in some other content 
- Ensure the editor updates
